regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -66,12 +66,55 @@ var CommentReviewView = Backbone.View.extend({
     this.$content.html(html);
     this.findElms();
 
+    this.initTabs();
+    this.initDependencies();
+
     this.$form.find('[name="comments"]').val(JSON.stringify(commentData));
 
     this.preambleHeadView = new PreambleHeadView();
     CommentEvents.trigger('comment:writeTabOpen');
   },
 
+  initTabs: function() {
+    function updateTabs(tab) {
+      $('[data-tab]').removeClass('current');
+      $('[data-tab="' + tab + '"]').addClass('current');
+      $('[data-tabs]').each(function(idx, elm) {
+        var $elm = $(elm);
+        var tabs = $elm.data('tabs');
+        if (tabs.indexOf(tab) !== -1) {
+          $elm.show();
+        } else {
+          $elm.hide();
+        }
+      });
+    }
+    var $tabs = $('[data-tab]');
```

Comment:
Heads up that this assumes there can only be one set up tabs on the page. What do you think about providing an identifier for the "tab set" (or similar)?